web/crux/src/shared/user-access.guard.ts

```diff
@@ -20,19 +25,22 @@ export default abstract class UserAccessGuard implements CanActivate {
     }
 
     const request = context.getArgByIndex<IdRequest>(0)
-    if (!request.accessedBy) {
-      throw new InvalidArgumentException({
-        message: 'Missing property',
-        property: 'accessedBy',
-      })
-    }
-
     if (!request.id) {
       return true
     }
 
-    return this.canActivateWithIdRequest(request)
+    const metadata = context.getArgByIndex<Metadata>(1)
+    const cookie = metadata.getMap().cookie as string
+    if (!cookie) {
+      throw new Error('Cookie missing')
```

Comment:
Throw the deleted Exception or NotFound would be better. 

```throw new InvalidArgumentException({
        message: 'Missing property',
        property: 'accessedBy',
      })```

Reply:
Unauthorized please